examples/model-webhook-slash.rs

```diff
@@ -66,10 +62,9 @@ where
     // Extract the signature to check against.
     let signature = if let Some(hex_sig) = req
         .headers()
-        .get("x-signature-ed25519")
-        .and_then(|v| v.to_str().ok())
+        .get(twilight_util::signature_validation::SIGNATURE_HEADER)
```

Comment:
Not a comment directly to this, but I cannot attach a comment to the timestamp extraction above.
Timestamp extraction should also use the const from `signature_validation`: `TIMESTAMP_HEADER`